src/styles/navbar/navbar.css

```diff
@@ -1,76 +1,309 @@
-@import url('https://fonts.googleapis.com/css2?family=Oxygen&display=swap');
-
-.side-nav {
-  width: 100%;
-  height: 100vh;
-  background: #3a59fd;
-  font-family: 'Oxygen', sans-serif;
-  color: white;
-  text-align: left;
-}
-
-.side-nav a {
-  color: white;
-}
-
-.side-nav a:hover {
-  color: white;
-  text-decoration: none;
-}
-
-ul {
-  list-style: none;
-  margin-left: -20px;
-}
-
-li {
-  cursor: pointer;
-  font-size: 12px;
-  font-weight: bold;
-}
-
-.user-image {
-  padding-top: 20px;
-}
-
-.user-image img {
-  width: 40px;
-}
-
-.welcome {
-  padding-top: 20px;
-}
-
-.username {
-  font-weight: bold;
-  line-height: 2;
-}
-
-.side-routes {
-  padding-top: 50px;
-}
-
-.route {
-  padding-top: 25px;
-}
-
-.icon {
-  background: #497bfe;
-  width: 25px;
-  padding: 6px;
-  border-radius: 5px;
-}
-
-.routes {
-  display: flex;
-}
-
-.name {
-  padding-left: 10px;
-  padding-top: 2px;
-}
-
-.logout {
-  position: absolute;
-  bottom: 50px;
-}
+/*--------------------------------------------------------------
+	## Header
+--------------------------------------------------------------*/
+.header-area {
+  padding: 0 0 98px;
+  position: relative; }
+  .header-area .banner-slider-item {
+    height: 96vh; }
+  .header-area .header-inner-details {
+    height: 100vh;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5568900)